misc/flatpak/README.asciidoc

```diff
@@ -0,0 +1,61 @@
+Flatpak
+=======
+
+The flatpak is based on the org.kde.Platform version 5.9 and ships all
```

Comment:
I think that we should put the commands needed to install the kde platform, just in case the user is new to flatpak (as I am)

I think `sudo flatpak install flathub org.kde.Sdk//5.9 and sudo flatpak install flathub org.kde.Platform//5.9` is the command we need?

Reply:
There's no need.  It will get pulled in as a dependency if it's not installed.

Reply:
Doesn't seem like it:

```
$ sudo flatpak install flathub org.kde.Sdk/x86_64/5.9
Installing: org.kde.Sdk/x86_64/5.9 from flathub
[####################] 21 delta parts, 203 loose fetched; 481063 KiB transferred in 126 seconds
Installing: org.freedesktop.Platform.VAAPI.Intel/x86_64/1.6 from flathub
[####################] 1 delta parts, 2 loose fetched; 2336 KiB transferred in 1 seconds
Installing: org.freedesktop.Platform.ffmpeg/x86_64/1.6 from flathub
[####################] 1 delta parts, 2 loose fetched; 2185 KiB transferred in 1 seconds
Installing: org.kde.Sdk.Locale/x86_64/5.9 from flathub
[####################] 5 delta parts, 128 loose fetched; 99863 KiB transferred in 17 seconds
```

```
$ flatpak-builder --force-clean app-dir org.qutebrowser.qutebrowser.json
error: org.kde.Platform 5.9 not installed
Failed to init: Unable to find runtime org.kde.Platform version 5.9
```